scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/sop.yaml

```diff
@@ -2,6 +2,31 @@ FileHeader:
   library_Suffix: 'SO'
   device_type: 'SOP'
 
+
+SOP-8_4.14x5.33mm_P1.27mm:
+  size_source: 'https://ww2.minicircuits.com/case_style/XX211.pdf'
+  body_size_x:
+    nominal: 4.14
+    tolerance: 0.762
```

Comment:
All dimensions are given with 3 significant digits (imperial dimensions) meaning the tolerance should be 0.015 inch (=0.381mm)

Additionally: I think body dimensions are all given as maximums in the table. (meaning you would need to subtract the tolerance from the maximum to reach the nominal size.)

Reply:
This annoying Mini-Circuits datasheets...
Yes, you're probably right about using the 0.015 inch tolerance. But I think they are the nominal dimensions as the tolerance is specified as +- 0.0015 inches.

Reply:
ah, sorry. For the body you're right, there it is specified.

Reply:
changing this also changes the name of the footprint, as the nominal size changes, creating the rather ugly `SOP-8_3.759x4.949mm_P1.27mm.kicad_mod`. Should I keep it or change it somehow?

Reply:
I agree. This is also the main source of confusion in their data sheets. I will round it as you suggested and make a remark about it.